modules/testsuite/cxf-tests/src/test/java/org/jboss/test/ws/jaxws/cxf/clientConfig/CXFClientConfigElytronNotOnClasspathTestCase.java

```diff
@@ -0,0 +1,140 @@
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.wsf.stack.cxf.client.ProviderImpl;
+import org.jboss.wsf.test.JBossWSTest;
+import org.jboss.wsf.test.JBossWSTestHelper;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.StringTokenizer;
+
+@RunWith(Arquillian.class)
+public class CXFClientConfigElytronNotOnClasspathTestCase extends JBossWSTest {
+   private final String FS = System.getProperty("file.separator"); // '/' on unix, '\' on windows
+
+   @ArquillianResource
+   private URL baseURL;
+
+   public static final String CLIENT_JAR = JBossWSTestHelper.writeToFile(new JBossWSTestHelper.JarDeployment("jaxws-cxf-jbws3713-client.jar") {
+      {
+         archive
+                 .setManifest(new StringAsset("Manifest-Version: 1.0\n"
+                         + "Main-Class: org.jboss.test.ws.jaxws.cxf.clientConfig.CXFClientElytronNotOnClasspathMainClass\n"
+                         + "Dependencies: org.jboss.ws.cxf.jbossws-cxf-client services export,org.apache.cxf.ws-security services export,org.apache.ws.security,org.apache.cxf.impl,org.apache.cxf.services-sts,org.apache.cxf.ws-security,org.apache.cxf,org.apache.ws.xmlschema,org.apache.ws.security,org.apache.cxf.services-sts,org.apache.cxf,org.apache.cxf.ws-security,org.wildfly.security.elytron,org.jboss.ws.cxf.jbossws-cxf-client,org.apache.cxf.impl,org.jboss.ws.jaxws-client\n"))
+                 .addClass(org.jboss.test.ws.jaxws.cxf.httpauth.Hello.class)
+                 .addClass(org.jboss.test.ws.jaxws.cxf.httpauth.HelloImpl.class)
+                 .addClass(org.jboss.test.ws.jaxws.cxf.httpauth.HelloRequest.class)
+                 .addClass(org.jboss.test.ws.jaxws.cxf.httpauth.HelloResponse.class)
+                 .addClass(org.jboss.test.ws.jaxws.cxf.httpauth.ObjectFactory.class)
+                 .addAsResource(new File(JBossWSTestHelper.getTestResourcesDir() + "/jaxws/cxf/clientConfig/META-INF/jaxws-client-config.xml"), "META-INF/jaxws-client-config.xml")
+                 .addAsResource(new File(JBossWSTestHelper.getTestResourcesDir() + "/jaxws/cxf/httpauth/WEB-INF/wsdl/hello.wsdl"), "META-INF/wsdl/hello.wsdl")
+                 .addAsResource(new File("/home/skylar/work/projects/integrations/ws/jbossws-cxf/modules/testsuite/cxf-tests/src/test/resources/jaxws/cxf/clientConfig/META-INF/wildfly-config-http-basic-auth.xml"), "META-INF/wildfly-config.xml")
+                 .addAsManifestResource(new File(JBossWSTestHelper.getTestResourcesDir() + "/jaxws/cxf/jbws3713/WEB-INF/client-permissions.xml"), "permissions.xml")
```

Comment:
could we use own copy of that file here? so that we don't have to worry about future changes

Reply:
Fixed.